aws-iot-certificate/src/main/java/com/amazonaws/iot/certificate/UpdateHandler.java

```diff
@@ -53,16 +53,14 @@ public UpdateHandler(IotClient iotClient) {
             try {
                 proxy.injectCredentialsAndInvokeV2(updateRequest, iotClient::updateCertificate);
 
-            } catch (final CertificateStateException e) {
+            } catch (final CertificateStateException | ServiceUnavailableException e) {
```

Comment:
Same comment as for DeleteHandler. CertificateStateException should probably not be translated like this. Applies to other handlers.

Reply:
Translating this to `CfnInvalidRequestException`.